rules/integrations/cloud_defend/privilege_escalation_mount_launched_inside_a_privileged_container.toml

```diff
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2023/10/26"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/10/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the use of the mount utility from inside a privileged container. The mount command is used to make a
+device or file system accessible to the system, and then to connect its root directory to a specified mount point on the
+local file system. When launched inside a privileged container--a container deployed with all the capabilities of the
+host machine-- an attacker can access sensitive host level files which could be used for further privilege escalation
+and container escapes to the host machine. Any usage of mount inside a running privileged container should be further
+investigated.
+"""
+from = "now-6m"
+index = ["logs-cloud_defend*"]
+interval = "5m"
+language = "eql"
+license = "Elastic License v2"
+name = "Mount Launched Inside a Privileged Container"
+references = [
+    "https://book.hacktricks.xyz/linux-hardening/privilege-escalation/docker-security/docker-breakout-privilege-escalation#privileged",
+]
+risk_score = 73
+rule_id = "41f7da9e-4e9f-4a81-9b58-40d725d83bc0"
+severity = "high"
+tags = [
+    "Data Source: Elastic Defend for Containers",
+    "Domain: Container",
+    "OS: Linux",
+    "Use Case: Threat Detection",
+    "Tactic: Privilege Escalation",
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.module == "cloud_defend" and  event.type== "start" and 
+(process.name== "mount" or process.args== "mount") and container.security_context.privileged == true
```

Comment:
Do you think having the process.args "mount" in there might cause FPs? Wouldn't just process.name == "mount" suffice? And potentially add a `and process.args_count >= 3` in there to reduce FPs further?

Reply:
On the query front, may be a new rule or a diff rule [ you could take a call ] . We have a mount execution coupled with hidepid parameter detected on linux systems in rule "rules/linux/defense_evasion_mount_execution.toml".

We could check if this make sense for containers and add these as well.

Reply:
@Aegrah this data source doesn't include process.args_count which I considered, and with some container `mount` would be called via another wrapper utility like `busybox` and this datasource would then make the process.name `busybox` and the args would be `mount`. There is the potential for false positives because of this but I'm not sure which is worse, missing out on the use of `mount` inside of a privileged container because it's wrapped inside another utility or risking a few FPs where `mount` is a process arg.

Reply:
@shashank-elastic I'll look into that, thank you